### PR TITLE
Update the logging namespace - qemu (t)

### PR DESCRIPTION
**Pull request by luckyh**
Issue: https://github.com/autotest/tp-qemu/issues/3048

Signed-off-by: Xu Han <xuhan@redhat.com>

### CODE REVIEW

File: qemu/tests/timedrift_no_net.py [hunk 1]
```diff
@@ -21,13 +23,13 @@ def _get_session(self):
     @error_context.context_aware
     def action_during_suspend(self, **args):
         error_context.context("Sleep a while before resuming guest",
-                              logging.info)
+                              LOG_JOB.info)
```
Comment (yanan-fu) [inline qemu/tests/timedrift_no_net.py:26]: `class GuestSuspendBaseTest(utils_test.qemu.GuestSuspend)`, and `utils_test.qemu.GuestSuspend` use `self.test = test`. By definition,  `test` can be anything,   but as ` self.test.fail` be used in derived class `GuestSuspendNegativeTest(GuestSuspendBaseTest)`,   so i think replace it with `self.test.log` is acceptable too.

File: qemu/tests/timedrift_no_net.py [hunk 2]
```diff
@@ -36,10 +38,10 @@ def subw_guest_suspend(test, params, vm, session):
 
     suspend_type = params.get("guest_suspend_type")
     if suspend_type == gs.SUSPEND_TYPE_MEM:
-        error_context.context("Suspend vm to mem", logging.info)
+        error_context.context("Suspend vm to mem", LOG_JOB.info)
         gs.guest_suspend_mem(params)
     elif suspend_type == gs.SUSPEND_TYPE_DISK:
-        error_context.context("Suspend vm to disk", logging.info)
+        error_context.context("Suspend vm to disk", LOG_JOB.info)
```
Comment (yanan-fu) [inline qemu/tests/timedrift_no_net.py:44]: Can be replaced with `test.log` in this funtion

File: qemu/tests/timedrift_no_net_win.py [hunk 3]
```diff
@@ -16,13 +18,13 @@ def __init__(self, test, params, vm, session):
...
         error_context.context("Sleep a while before resuming guest",
-                              logging.info)
+                              LOG_JOB.info)
 
         time.sleep(float(self.params.get("wait_timeout", "1800")))
         if self.os_type == "windows":
             # Due to WinXP/2003 won't suspend immediately after issue S3 cmd,
             # delay 10~60 secs here, maybe there's a bug in windows os.
-            logging.info("WinXP/2003 need more time to suspend, sleep 50s.")
+            LOG_JOB.info("WinXP/2003 need more time to suspend, sleep 50s.")
```
Comment (yanan-fu) [inline qemu/tests/timedrift_no_net_win.py:27]: Same with `timedrift_no_net.py`

File: qemu/tests/timedrift_no_net_win.py [hunk 4]
```diff
@@ -31,10 +33,10 @@ def subw_guest_suspend(test, params, vm, session):
 
     suspend_type = params.get("guest_suspend_type")
     if suspend_type == gs.SUSPEND_TYPE_MEM:
-        error_context.context("Suspend vm to mem", logging.info)
+        error_context.context("Suspend vm to mem", LOG_JOB.info)
         gs.guest_suspend_mem(params)
     elif suspend_type == gs.SUSPEND_TYPE_DISK:
-        error_context.context("Suspend vm to disk", logging.info)
+        error_context.context("Suspend vm to disk", LOG_JOB.info)
```
Comment (yanan-fu) [inline qemu/tests/timedrift_no_net_win.py:39]: Can be replaced with `test.log` in this function.

**Comment by luckyh** [comment]
@yanan-fu have updated per your suggestion, thanks.

**Review by yanan-fu — APPROVED** [review]
LGTM

Acked-by: Yanan Fu <yfu@redhat.com>

**Pull request merged**